app/Http/Livewire/Components/Header.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Livewire\Components;
+
+use App\Http\Livewire\Traits\InteractWithModal;
+use App\Models\Configs;
+use Illuminate\Support\Facades\Config;
+use Livewire\Component;
+
+class Header extends Component
+{
+	use InteractWithModal;
+
+	/**
+	 * @var string
+	 */
+	public $title = '';
+
+	/**
+	 * @var string
+	 */
+	public $mode;
+
+	public function mount(?string $mode = 'albums', ?string $title = null)
+	{
+		$this->title = $title ?? Configs::get_value('site_title', Config::get('defines.defaults.SITE_TITLE'));
+		$this->mode = $mode ?? 'albums';
+	}
+
+	public function render()
+	{
+		return view('livewire.header.header');
+	}
+
+	public function login()
```

Comment:
```suggestion
	public function login(): void
```

Add PHPdoc comment. Why is `login()` a method of header? Can you login to a header?

Reply:
Rather than answering all those questions, I should rather try to create a bigger picture of how Livewire work.
In that I also hope to answer a lot of your questions. :)

I am not denying that I should document the code, but what appears to me is that you are missing the bigger picture of how this whole part works (it is not a complaint or a reproach). :)